.rubocop.yml

```diff
@@ -31,6 +32,11 @@ Layout/DotPosition:
 Layout/EndAlignment:
   EnforcedStyleAlignWith: variable
 
+#################### RSpec ###########################
+
+RSpec/NestedGroups:
```

Comment:
I don't mind having this in the PR, but it should be in a different commit. It's not related to the notification filters.

Reply:
Yes, I'll do it in another PR, because including RSpec cops is uncovering a lot of issues. I'm going back to having it disabled locally from that specific spec.